src/ASM/ASMbase.h

```diff
@@ -165,6 +165,8 @@ class ASMbase
   virtual unsigned char getNoFields(int b = 0) const { return b > 1 ? 0 : nf; }
   //! \brief Returns the number of Lagrange multipliers per node.
   unsigned char getNoLagPerNode() const { return nLag; }
+  //! \brief Obtain first and last node for multipliers.
+  std::pair<size_t, size_t> getLagrangeMultipliers() { return myLMs; }
 
```

Comment:
Denne metoden har jeg litt prinsippielle motforestillinger mot (understreker litt). Fordi her avsløres det for omverdenen at ASMbase lagrer info om hvilke noder som er av type Lagrange multiplier som en pair. Med det er implemetasjonsdetaljer egentlig. I DD-klassen der denne er brukt, bør man heller loope over alle noder i patchen og så bruke isLMn evt. getLMType der det ønskes å operere på disse nodene. Ja, man løkker da over noen flere noder enn nødvendig, men med mindre denne kostnaden er merkbar tror jeg det er verdt det, for modularitetshensyn (kan hende man en vakker dag finner på å legge LM-noder inne blandt øvrige noder).